tests/testsuite/doc.rs

```diff
@@ -1638,3 +1639,145 @@ fn crate_versions_flag_is_overridden() {
     p.cargo("rustdoc -- --crate-version 2.0.3").run();
     asserts(output_documentation());
 }
+
+#[cargo_test]
+fn doc_fingerprint_versioning_consistent() {
+    #[derive(Debug, Serialize, Deserialize)]
+    pub struct RustDocFingerprint {
+        pub rustc_verbose_version: String,
+    }
```

Comment:
This can probably just use the definition from the `cargo` crate.

Reply:
Could you expand a bit on this. I don't understand what you mean.

Reply:
Instead of redefining the struct, you should be able to just reference `cargo::core::compiler::RustDocFingerprint` directly from the test.